osi_object.proto

```diff
@@ -572,7 +572,37 @@ message MovingObject
         //
         // \note OSI uses singular instead of plural for repeated field names.
         //
-        repeated double assigned_lane_percentage = 2;        
+        repeated double assigned_lane_percentage = 2;
+
+        // Classification of the general moving direction of a moving object depeding on the relative yaw orientation between host vehicle and moving object.
```

Comment:
depending

Reply:
Suggestion:
// Classification of the general moving direction of a moving object depeding on the relative yaw orientation to the host vehicle.
And: what if hostvehicle_id is not set?

Reply:
@ThomasSchloemicherAVL please comment